taxonomies/labels.txt

```diff
@@ -11,22 +11,26 @@ stopwords:sv:certifierad, ingrediens, godkänd, råvara, enlighet, med, kriterie
 synonyms:en:coloring, colouring, color, colour
 synonyms:ru:окраска, цвет
 synonyms:fi:väri, värjäys, väriaine
+synonyms:fr:colorant, couleur
 synonyms:es:color, colorante
 
 synonyms:en:flavor, flavour, flavoring, flavouring
 synonyms:fi:aromi, maku
+synonyms:fr:saveur, arôme
```

Comment:
Same here, unless we have examples of labels where saveurs is used instead of arômes, we should not add it.